cmd/watch.go

```diff
@@ -79,6 +79,14 @@ func watch(ctx *cmdutil.Ctx, events chan file.Event, sig chan os.Signal) error {
 func perform(ctx *cmdutil.Ctx, path string, op file.Op) {
 	defer ctx.DoneTask()
 
+	if op == file.Skip {
+		// TODO: which colour for Skipped files?
```

Comment:
@attaboy Any thoughts here? Do we need to distinguish skipped files from non-skipped?

Reply:
This will be addressed in https://github.com/Shopify/themekit/issues/727